pkg/agent/openflow/fields.go

```diff
@@ -123,6 +123,13 @@ var (
 	TrafficControlRedirectRegMark = binding.NewRegMark(TrafficControlActionField, 0b10)
 	// reg4[24]: Mark to indicate that whether the Service is backed by Service IPs of other Services.
 	NestedServiceRegMark = binding.NewOneBitRegMark(4, 24)
+	// reg4[25]: Mark to indicate that whether the Service traffic works in DSR mode.
+	DSRServiceRegMark    = binding.NewOneBitRegMark(4, 25)
+	NotDSRServiceRegMark = binding.NewOneBitZeroRegMark(4, 25)
+	// reg4[26]: Mark to indicate that whether a remote non-hostNetwork Endpoint is selected.
+	RemoteEndpointRegMark = binding.NewOneBitRegMark(4, 26)
+	// reg4[27]: Mark to indicate that whether the traffic's source is an external IP.
```

Comment:
Can the IPs on a K8s Node, excluding antrea-gw0, be considered external IPs here?